### DISCUSSION TITLE
VM uses a lot more storage than whats dedicated

### DISCUSSION THREAD

**Discussion by amirseyedian** [utmapp/UTM#5185] [UTM Help (macOS)] [answered]
**Issues Description**  
I have specified 64GB of storage upon creation for a vm but now it consumes about twice that amount for no apparent reason everything else seem normal and behaves as expected.

**Configuration**  
* UTM Version: 4.0.8/67
* macOS Version: 13.2.1
* Mac Chip (Intel, M1, ...): M1

Storage Status in VM:
![Screenshot 2023-04-11 at 10 24 30 cleaned](https://user-images.githubusercontent.com/105858041/231071628-cee0364c-40f4-4277-8cb3-a277f95a82fd.png)

![Screenshot 2023-04-11 at 10 24 08 cleaned](https://user-images.githubusercontent.com/105858041/231071470-7e951c78-1c7b-4309-af63-231eae2a74da.png)

The amount of storage it consumes on host/mac
![Screenshot 2023-04-11 at 10 30 59 cleaned](https://user-images.githubusercontent.com/105858041/231071811-a2e1cbcd-bfab-420c-8b9a-5e7cd71ff525.png)
![Screenshot 2023-04-11 at 10 32 01 cleaned](https://user-images.githubusercontent.com/105858041/231071842-fb50f9d7-7faa-49af-9251-2aadb3140599.png)

* Plist file
[https://paste.debian.net/1276992/](url)

Im aware of the **Reclaim Space** and **Compress** in configuration however Im not familiar with their functionality and rather proceed with caution since I dont have a backup yet

**Comment by LostInCompilation** [comment, marked as answer]
That's normal, since free space once used by guest appear as GBs of zeros on host. You need to reclaim disk space (and compress if you want).
